modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/TranslationOptionsContainer.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayAlert from '@clayui/alert';
+import {ClayToggle} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClayLink from '@clayui/link';
+import {ClayTooltipProvider} from '@clayui/tooltip';
+import {Card} from '@liferay/object-js-components-web';
+import React from 'react';
+
+import './TranslationOptionsContainer.scss';
+
+interface TranslationOptionsContainerProps {
+	isApproved: boolean;
+	setValues: (values: Partial<ObjectField>) => void;
+	values: Partial<ObjectField>;
+}
+
+export function TranslationOptionsContainer({
+	isApproved,
+	setValues,
+	values,
+}: TranslationOptionsContainerProps) {
+	const translatableField =
+		(values.businessType === 'LongText' ||
+			values.businessType === 'Text' ||
```

Comment:
@danvb112 Please, sort the conditions for businessType in alphabetical order. Thanks :)